src/codex_plus/main_sync_cffi.py

```
@@ -241,6 +738,20 @@ async def proxy(request: Request, path: str):
 
+    # ✅ SAFE TO MODIFY: Agent orchestrator middleware processing
+    # Process agent invocation patterns (/agent, /agents run, /delegate)
+    try:
+        logger.info(f"🤖 Processing agent commands for {path}")
+        modified_request_data = await agent_middleware.process_request(request, path)
+        if modified_request_data:
+            # Agent commands were processed, update request body
+            modified_body = json.dumps(modified_request_data).encode('utf-8')
+            request.state.modified_body = modified_body
+            logger.info("🤖 Agent commands processed, request body modified")
+    except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)